network/views.py

```diff
@@ -10,27 +11,33 @@
 from .models import User, Post, UserFollowing
 
 
-class PostForm(ModelForm):
-    class Meta:
-        model = Post
-        fields = ['content']
-
-
 def index(request):
-    return render(request, "network/index.html", {
-        'form': PostForm(), 
-    })
+    return render(request, "network/index.html")
+
 
+def post_list(request, feed):
```

Comment:
i think its more important to unit test this logic
you can do it through using mocks or other tools
https://stackoverflow.com/questions/2036202/how-to-mock-users-and-requests-in-django